src/main/javascript/crypto/src/eas/EasAttestedObject.ts

```diff
@@ -0,0 +1,112 @@
+import {EasTicketAttestation, TicketSchema} from "./EasTicketAttestation";
+import {EASIdentifierAttestation} from "./EASIdentifierAttestation";
+import {AttestationCrypto} from "../libs/AttestationCrypto";
+import {ProofOfExponentInterface} from "../libs/ProofOfExponentInterface";
+import {hexStringToUint8, logger, uint8tohex} from "../libs/utils";
+import {AsnParser, AsnProp, AsnPropTypes, AsnSerializer} from "@peculiar/asn1-schema";
+import {KeyPair, KeysArray} from "../libs/KeyPair";
+import {UsageProofOfExponent} from "../libs/UsageProofOfExponent";
+import {DEBUGLEVEL} from "../config";
+
+export class EasUseToken {
```

Comment:
would be great to move this ASN class to the file in /asn1 folder to they them in one place

Reply:
I'd rather keep it here or as you said below, move to fully abi encoded structure.